test/Hpack/ConfigSpec.hs

```diff
@@ -642,6 +642,14 @@ spec = do
         }
         )
 
+    it "accepts setup-depends" $ do
+      withPackageConfig_ [i|
+        setup-depends:
+          - foo-1.0
+          - bar-2.0
+        |]
+        (packageSetupDepends >>> (`shouldBe` ["foo-1.0", "bar-2.0"]))
```

Comment:
I would assume this has to follow the same format as other dependencies, say `foo == 1.0` instead of `foo-1.0`?